pandas/tests/test_base.py

```diff
@@ -683,6 +683,10 @@ def test_factorize(self):
 
     def test_duplicated_drop_duplicates(self):
         # GH 4060
+
+        import warnings
+        warnings.simplefilter('always')
```

Comment:
u will have lots of failures 

need to either remove some of the tests that have take_last or convert them to keep (prob best) or use an assert produces around them